medusa/server/api/v2/base.py

```diff
@@ -420,10 +425,9 @@ def patch(self, target, value):
                     setattr(target, self.attr, self.converter(value))
             except AttributeError:
                 log.warning(
-                    'Error trying to change attribute %s on target %s, you sure'
-                    ' you are allowed to change this attribute?',
-                    self.attr,
-                    target
+                    'Error trying to change attribute {attr} on target {target}'
```

Comment:
I think `{target!r}` is better since it's usually an object.
For example when patching as series it would show up as:
```
Error trying to change attribute status on target indexerid: 1234
indexer: 1
name: Show Name
```
etc...